Pico8Emulator/unit/audio/AudioUnit.cs

```diff
@@ -97,7 +97,8 @@ public class AudioUnit : Unit {
 		}
 
 		public void Music(int n, int? fade_len = null, int? channel_mask = null) {
-			_musicPlayer.Start(n);
+			// This is broken, so commented out for now
+			//_musicPlayer.Start(n);
```

Comment:
This shouldn't be commented out since it is useful to check when it doesn't work. It still works in some cases so I'd like to keep it running so I know when it actually fails.

Reply:
The thing is, this crashes the carts, that call music()

Reply:
but that's good, right ? Now we know that there's a problem we need to fix, which we wouldn't have if it was commented out before.

I already found the bug, will create a PR to fix it